compose.example.yml

```diff
@@ -0,0 +1,21 @@
+---
+services:
+  sml2mqtt:
+    image: spacemanspiff2007/sml2mqtt
+    container_name: sml2mqtt
+    init: true
+    restart: unless-stopped
+    mem_limit: 64m
```

Comment:
Can you add a comment that this limits the docker memory and that it is sufficient for you. And a suggestion to comment out in case the user is unsure.